gamemodes/terrortown/gamemode/client/cl_voice.lua

```diff
@@ -53,7 +55,7 @@ end
 local function VoiceTeamTryEnable()
 	local client = LocalPlayer()
 
-	if not hook.Run("TTT2CanUseVoiceChat", client, true) then return false end
+	if hook.Run("TTT2CanUseVoiceChat", client, true) == false then return false end
```

Comment:
Return false in new line. I didn‘t mentioned it before because I thought you will look on your own into these files so that I don‘t need to spam ^^ nvm :p

Reply:
This wasn't my code :P